internal/srv/app.go

```diff
@@ -210,27 +210,27 @@ func (s *Server) removeDeployment(ctx context.Context, lb *loadBalancer) error {
 
 	client, err := s.newHelmClient(hash)
 	if err != nil {
-		s.Logger.Errorw("unable to initialize helm client", "error", err, "loadBalancer", lb.loadBalancerID.String(), "namespace", hash, "releaseName", releaseName)
+		s.Logger.Debugw("unable to initialize helm client", "error", err, "loadBalancer", lb.loadBalancerID.String(), "namespace", hash, "releaseName", releaseName)
 		return err
 	}
 
 	hc := action.NewUninstall(client)
 	_, err = hc.Run(releaseName)
 
 	if err != nil {
-		s.Logger.Errorw("unable to remove loadBalancer", "error", err, "releaseName", releaseName, "loadBalancer", lb.loadBalancerID.String(), "namespace", hash)
+		s.Logger.Debugw("unable to remove loadBalancer", "error", err, "releaseName", releaseName, "loadBalancer", lb.loadBalancerID.String(), "namespace", hash)
```

Comment:
Isn't this something we wanna see? I think adding and removing stuff should be Info logged. We'll never really see Debug logs except when running locally.